monitor/README.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: pattern
+title: Monitor
+folder: monitor
+permalink: /patterns/monitor/
+categories: Concurrency
+tags:
+ - Thread-safe
+ - Mutual exclusion
```

Comment:
The existing categories and tags can be found at https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags. They should be used unless you have a very good reason to invent to new ones.

Reply:
i have changed to Performance